crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -96,6 +97,22 @@ struct Opts {
     /// gateway_config_path, keypair_path and primary_gas_id
     #[clap(long, parse(try_from_str), default_value = "true", global = true)]
     pub local: bool,
+    // Default workload is 100% transfer object
+    #[clap(subcommand)]
+    workload_spec: OptWorkloadSpec,
+}
+
+#[derive(Debug, Clone, Parser, Eq, PartialEq, EnumString)]
+#[clap(rename_all = "kebab-case")]
+pub enum OptWorkloadSpec {
+    WorkloadSpec {
+        // shared counter workload weight
+        #[clap(long, default_value = "0")]
+        shared_counter: u32,
+        // transfer object workload weight
+        #[clap(long, default_value = "1")]
+        transfer_object: u32,
```

Comment:
Instead of "workload weight" here, I would say "relative weight in the benchmark workload". This hints at the mechanics of the mixing more precisely.

Reply:
done

Reply:
done